Backend/.env.example

```diff
@@ -1,6 +1 @@
-DB_USER=root
-DB_HOST=localhost
-DB_PASSWORD=root
-DB_NAME=task
-DB_DIALECT=mysql
-PORT=5000
+DATABASE_URL="sqlserver://SIDDHESH:1443;database=task;user=sa;password=root;encrypt=true;trustServerCertificate=true"
```

Comment:
Generally, in .env we specify config which will be set by users. So keeping whole DATABASE_URL will not work. So keep separate variables such as 
1. host - DB_HOST 
2. database name - DB_NAME
3. user - DB_USER
4. password - DB_PASSWORD 

And the form a DB_CONNECTION_SCHEMA in code.